Makefile

```diff
@@ -66,11 +66,9 @@ dev: DEBUG = desktop:*
 dev: 
 	$(MAKE) start NODE_ENV=$(NODE_ENV) DEBUG=$(DEBUG)
 
-
-BASE_CONFIG := $(THIS_DIR)/desktop-config/config-base.json
-ENV_CONFIG := $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
-
 .PHONY: desktop$/config.json
+desktop$/config.json: BASE_CONFIG = $(THIS_DIR)/desktop-config/config-base.json
+desktop$/config.json: ENV_CONFIG = $(THIS_DIR)/desktop-config/config-$(CONFIG_ENV).json
 desktop$/config.json:
 ifeq (,$(wildcard $(ENV_CONFIG)))
```

Comment:
@ockham I initially had this approach in #467 but it turned out that `ifeq (,$(wildcard $(ENV_CONFIG)))` never evaluates to true.

Running this running results in:
```console
adlk@work: $ make build CONFIG_ENV=release
Makefile:73: Config file for environment "release" does not exist. Ignoring environment.
```